libs/designsystem/src/lib/components/modal/modal-wrapper/modal-wrapper.component.ts

```diff
@@ -38,48 +40,42 @@ export class ModalWrapperComponent implements AfterViewInit, OnDestroy {
   }
 
   constructor(
-    params: NavParams,
-    private modalController: IModalController,
-    injector: Injector,
+    private injector: Injector,
     private elementRef: ElementRef<HTMLElement>,
     private renderer: Renderer2
-  ) {
-    this.config = params.get('config');
+  ) {}
+
+  ngOnInit(): void {
     this.componentPropsInjector = Injector.create({
       providers: [{ provide: COMPONENT_PROPS, useValue: this.config.componentProps }],
-      parent: injector,
+      parent: this.injector,
     });
-    this.registerScrolling(this.config.modal);
   }
 
   ngAfterViewInit(): void {
     this.checkForEmbeddedFooter();
   }
 
-  private registerScrolling(modal: Modal) {
-    modal.scrollToTop = this.scrollToTop.bind(this);
-    modal.scrollToBottom = this.scrollToBottom.bind(this);
-  }
-
-  private scrollToTop(scrollDuration?: KirbyAnimation.Duration) {
+  public scrollToTop(scrollDuration?: KirbyAnimation.Duration) {
     this.ionContent.scrollToTop(scrollDuration || 0);
   }
 
-  private scrollToBottom(scrollDuration?: KirbyAnimation.Duration) {
+  public scrollToBottom(scrollDuration?: KirbyAnimation.Duration) {
     this.ionContent.scrollToBottom(scrollDuration || 0);
   }
 
+  public close(data?: any) {
```

Comment:
Consider consistency with other method's access modifiers, either prefix with `public` or not.

Reply:
Good point, fixed 👍